src/App.js

```diff
@@ -1,23 +1,61 @@
 import './App.css';
 import Form from "./components/Form/Form"
-import {useState} from "react"
+import {useEffect, useState} from "react"
 import {uid} from "uid"
 import List from "./components/List/List"
 import useLocalStorageState from "use-local-storage-state";
 
 function App() {
    const [activities, setActivities] = useLocalStorageState("activities", {defaultValue: []});
+   const [weather, setWeather] = useState()
+   const [temperature, setTemperature] = useState()
+   const [condition, setCondition] = useState()
 
    function handleAddActivity(newActivity) {
       setActivities([{...newActivity, id: uid()}, ...activities])
    }
+   
+   const goodWeatherActivities = activities.filter((activity) => activity.isForGoodWeather === true)
+   const badWeatherActivities = activities.filter((activity) => activity.isForGoodWeather === false)
```

Comment:
you actually don't need two different variables for this:

this should work as well:
```js
const filteredActivities = activities.filter((activity) => activity.isForGoodWeather === weather)
```